test/services/billing/supplementary/fetch-invoices-to-be-reissued.service.test.js

```diff
@@ -30,6 +31,15 @@ describe('Fetch Invoices To Be Reissued service', () => {
     billingInvoice = await BillingInvoiceHelper.add({ billingBatchId: billingBatch.billingBatchId })
     const { billingInvoiceLicenceId } = await BillingInvoiceLicenceHelper.add({ billingInvoiceId: billingInvoice.billingInvoiceId })
     await BillingTransactionHelper.add({ billingInvoiceLicenceId })
+
+    // Set up a separate billing batch which we will manually patch to be alcs
+    const alcsBillingBatch = await BillingBatchHelper.add()
```

Comment:
Don't think we need line 42 if we write line 36 as `const alcsBillingBatch = await BillingBatchHelper.add({ scheme: 'alcs' })`